.scala-steward.conf

```diff
@@ -1,8 +1,9 @@
+commits.message = "bump: ${artifactName} ${nextVersion} (was ${currentVersion})"
 pullRequests.frequency = "@monthly"
 
 updates.pin  = [
   { groupId = "org.scalatest", artifactId = "scalatest", version = "3.1." }
-  { groupId = "com.fasterxml.jackson.core", artifactId = "jackson-core", version = "2.11." }
+  { groupId = "com.fasterxml.jackson.core", artifactId = "jackson-core", version = "2.13." }
```

Comment:
It would be useful to remove the `ignore` of databind because security micro patches are often added there, and also pin databind to 2.13